catalyst_voices/apps/voices/lib/pages/campaign/details/campaign_details_dialog.dart

```diff
@@ -0,0 +1,67 @@
+import 'dart:async';
+
+import 'package:catalyst_voices/pages/campaign/details/widgets/campaign_header.dart';
+import 'package:catalyst_voices/pages/campaign/details/widgets/campaign_sections_list_view.dart';
+import 'package:catalyst_voices/widgets/widgets.dart';
+import 'package:catalyst_voices_blocs/catalyst_voices_blocs.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+class CampaignDetailsDialog extends StatefulWidget {
+  final String id;
+
+  const CampaignDetailsDialog._({
+    required this.id,
+  });
+
+  static Future<void> show(
```

Comment:
You can trigger this dialog already from discovery_page.dart, I left a TODO there:
```
// TODO(dtscalac): show campaign details dialog
```